README.md

```diff
@@ -140,8 +169,8 @@ Once those dependencies are installed, run `bundle install` to install required
 
 To run the application locally:
 
-1. Run `yarn` to install dependencies for the web app to run
-2. Run `foreman start` to launch the app on <http://localhost:3000>
+1. Run `bin/setup` to install dependencies for the web app to run
+2. Run `bin/dev` to launch the app on <http://localhost:3000> or <https://apply-manage.localhost>.
```

Comment:
For additional context I run both of these every time I pull down the repo - it keeps everything update for me 👌